# Notes on Ebola outbreak, by paper

[@Frieden:2014fv]
- A commentary on the controls needed for prevention.
- Noted that PCR has become widely used for testing.

[@Gire:2014fk]
- Report on Sierra Leone and Guinea (2014) outbreak sequences.
- Use of genomic sequencing allowed reconstruction virus geographic movement (phylogeography).
- Identified many mutations in the genome, but no conclusions on their effects, only speculation on their role as determinants of pathogenicity and therapeutic avenues.

[@Park:2015cw]
- Further analysis of data from Sierra Leone outbreak, led by Danny!
- Phylogeographic analysis showed no evidence of cross-border transmission
- Intra-host variants were transmitted from patient to patient, indicating large infectious doses
- So-called deleterious mutations are removed from viral population over long periods of time - assumed that non-synonymous mutations are the indicator of deletrious mutations. This claim needs to be experimentally tested.

[@Baize:2014gx]
- Report on Zaire Ebola virus disease in Guinea
- Only 3 patient samples were sequenced. They were chosen to be sequenced because of high vRNA levels.
  - This is likely a challenge that needs to be overcome.
  - Sequences were determined using Sanger sequencing.
- Phylogenetic analysis showed that this was most related to the Zaire Ebola virus.

[@Gatherer:2014fp]
- A review article on the 2014 Ebola virus outbreak. Contains a timeline that might be useful to go back to in the future.
- Contains a table of genetic divergences. Not sure what the author is trying to convey there.
- Notes that Bayesian phylogenetic analyses suggest that spillover came from bats into humans. This is based on 2006 and 2005 studies.

[@Quick:2016kf]
- Published two years after the outbreak began; sample collection between Mar-Oct 2015.
- Showed the use of MinION sequencers.
- Authors rightly claim that real-time genomic surveillance has the potential to "improve the efficiency of resource allocation and the timeliness of epidemiological investigations", but do not clearly state how this could happen.

Overall:
- Genome sequencing gets paired most with phylogenetic analysis, because this tool is the most mature for that purpose.
- Phylogenetic analyses can tell us:
  - where a virus came from (phylogeography),
  - when it likely emerged (dating of tree nodes with molecular clock assumption) and
  - how it is transmitted (whether intra-host variation is transmitted),
- But cannot tell us how that virus may be dangerous. The above three are the current state-of-the-art; the latter is where we would like to be.
