[@Quick:2015ko]
- showed the use of a rapid MiSeq protocol on salmonella outbreak.
- showed the use of MinION as well - able to do "species-level identification" within minutes, and within 2 hours can determine whether a pathogen is part of an outbreak or not.

[@Ashton:2015ff]
- used MinION to generate long reads of a salmonella genome
- because of long reads, MinION is better than short-reads for resolving the genomic position of antibiotic resistance islands; short reads from whole genomes may result in antibiotic resistance genes being identified, but as being part of another genome.

[@denBakker:2014ke]
- investigated the practicality of whole genome sequencing in an outbreak setting
- throughput is on the order of approx. 10^1^-10^2^ isolates per day.
- argued that much of the pipeline, including "uploading, analysis, and reporting" can be automated.

[@Leekitcharoenphon:2016cf]
- Sequenced 315 S. Typhimurium DT104 isolates sampled from 1969 to 2012 from 21 countries on six continents.
- Bayesian reconstruction identified time of divergence and effective population size over time (skyline plot) of the global salmonella pool.
- There's a known "13-kb SGI1 multidrug resistance region" responsible for multiple drug resistance. Susceptible isolates lacked this region.
- Able to map the spread of drug resistance genes over space and time.

[@Wong:2015bi]
- sequenced MDR *S. Typhi* H58 lineage.
- phylogeographic analysis showed spread of antibiotic resistance genes, including those of ampicillin, trimethoprim, sulfonamides, chloramphenicol, streptomycin.
- Learned that Quinolone (e.g. ciprofloxacine) resistance can be mediated by **non-synonymous mutations** in quinolone-resistance determining regions. Used presence of non-synonymous mutations to infer that selection was ongoing. (note: this is quite similar to the logic used for ebola)

[@Struelens:2013uo]
- WOW, this editorial essay is a really good one.
- editorial essay on molecular and genomic epidemiology
- Whole genome sequencing is the main thing that they highlight at the end.
- authors claim that WGS has two key advantages:
  - provides maximal strain discrimination (oh yeah!)
  - can be linked to "clinically and epidemiologically relevant phenotypes"
- Challenges for WGS include:
  - speed
  - data analysis
  - interpretation: given the fast evolutionary processes that occur, the key challenge here will be the interpretation of amino acid polymorphisms and their effect on these clinically and epidemiologically relevant phenotypes
  - cost
- Author brings up the concept of a **"resistome"** (the set of genes involved in antimicrobial resistance) and the **"virulome"** (the set of genes linked to the virulence of an isolate).


Overall:
- What was a key assumption across the studies listed here? The link between harbouring the MDR locus and drug resistance was clear, established over multiple studies.
  - I personally think this is plausible/reasonable for antibiotic resistance, given its long history of research, but it is less clear for other aspects of the pathogenic capacity of a pathogen.
-

http://www.cdc.gov/nationalsurveillance/salmonella-surveillance.html
- CDC website on salmonella surveillance
- salmonella surveillance is governed by "serotype" identification
-

http://www.cdc.gov/salmonella/reportspubs/surveillance.html
- this website shows the networks of groups that report salmonella outbreaks.
- Laboratory-based Enteric Disease Surveillance (LEDS)
  - "Collects laboratory data, such as serotype, on Salmonella
- National Notifiable Diseases Surveillance System (NNDSS)
  - "Tracks notifiable infectious diseases across the US"
- Foodborne Disease Active Surveillance Network (FoodNet)
  - "Reports trends in foodborne infections and tracks the impact of food safety policies nationally"
- National Molecular Subtyping Network for Foodborne Disease Surveillance (PulseNet)
  - "Connects cases of illness nationwide to quickly identify outbreaks, including many that would otherwise go undetected"
  - uses Pulse Field Gel Electrophoresis, not genome sequencing
- National Antimicrobial Resistance Monitoring Systemâ€”enteric bacteria (NARMS)
  - "Tracking trends in resistance"
  - probably uses culture-based methods to identify drug resistance
- Foodborne Disease Outbreak Surveillance System (FDOSS)
  - Captures outbreak data on agents, foods, and settings responsible for illness
- Outbreak Response Team


http://www.genomicepidemiology.org
